[CI] EsqlActionBreakerIT testTopNPushedToLucene failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #10269 / almalinux-8-aarch64_checkpart3_platform-support-arm](https://gradle-enterprise.elastic.co/s/s6ryzwpscjads)
- [elasticsearch-pull-request #87594 / part-3](https://gradle-enterprise.elastic.co/s/q6mb3mc65asgq)
- [elasticsearch-pull-request #87233 / part-3](https://gradle-enterprise.elastic.co/s/4wiz6h4xypgiw)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:internalClusterTest" --tests "org.elasticsearch.xpack.esql.action.EsqlActionBreakerIT.testTopNPushedToLucene" -Dtests.seed=6643125721DEBCC8 -Dtests.locale=nds -Dtests.timezone=Pacific/Apia -Druntime.java=24
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.action.EsqlActionBreakerIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testTopNPushedToLucene'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Request breaker not reset to 0 on node: node_s1
Expected: <0L>
     but: was <-216L>
```

**Issue Reasons:**
- [main] 3 failures in test testTopNPushedToLucene (0.4% fail rate in 809 executions)
- [main] 2 failures in step part-3 (0.6% fail rate in 315 executions)
- [main] 2 failures in pipeline elasticsearch-pull-request (0.6% fail rate in 314 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -390,9 +390,6 @@ tests:
 - class: org.elasticsearch.packaging.test.DockerTests
   method: test050BasicApiTests
   issue: https://github.com/elastic/elasticsearch/issues/120911
-- class: org.elasticsearch.xpack.esql.action.EsqlActionBreakerIT
-  method: testFromEvalStats
-  issue: https://github.com/elastic/elasticsearch/issues/131503
 - class: org.elasticsearch.xpack.search.CrossClusterAsyncSearchIT
   method: testCancellationViaTimeoutWithAllowPartialResultsSetToFalse
   issue: https://github.com/elastic/elasticsearch/issues/131248
@@ -618,12 +615,6 @@ tests:
 - class: org.elasticsearch.test.rest.yaml.RcsCcsCommonYamlTestSuiteIT
   method: test {p0=search.vectors/90_sparse_vector/Indexing and searching multi-value sparse vectors in >=8.15}
   issue: https://github.com/elastic/elasticsearch/issues/133508
-- class: org.elasticsearch.xpack.esql.action.EsqlActionBreakerIT
-  method: testTopNPushedToLucene
-  issue: https://github.com/elastic/elasticsearch/issues/133556
-- class: org.elasticsearch.xpack.esql.action.EsqlActionBreakerIT
-  method: testFilterNestedFields
-  issue: https://github.com/elastic/elasticsearch/issues/133557
 - class: org.elasticsearch.test.rest.yaml.RcsCcsCommonYamlTestSuiteIT
   method: test {p0=search/10_source_filtering/no filtering}
   issue: https://github.com/elastic/elasticsearch/issues/133561

